pages/cookbook/gpt4-vision.livemd

```diff
@@ -33,6 +37,7 @@ base64_image = "data:image/png;base64," <> Base.encode64(image)
 
 defmodule Product do
   use Ecto.Schema
+  use Instructor.Validator
```

Comment:
why? there aren't any validations in this example?

Reply:
Ah. that's why! I thought it was necessary to `use Instructor.Validator`. Now i know better.